src/Compilers/Core/Portable/Operations/IFieldReferenceExpression.cs

```diff
@@ -17,6 +17,14 @@ public interface IFieldReferenceExpression : IMemberReferenceExpression
         /// Referenced field.
         /// </summary>
         IFieldSymbol Field { get; }
+        /// <summary>
+        /// If the field reference is also where the field was declared.
+        /// </summary>
+        /// <remarks>
+        /// This is only ever true in CSharp scripts, where a top-level statement creates a new variable
+        /// in a reference, such as an out variable declaration or a tuple deconstruction declaration.
```

Comment:
>tuple [](start = 69, length = 5)

Consider removing the "tuple", just "deconstruction declaration".

Reply:
Fixed.